### PR TITLE
docs: no more bricks

### PR DESCRIPTION
**Pull request by MthwRobinson**
### Summary

We no longer use the "bricks" terminology for partioning functions, etc in the library. This PR updates various references to bricks within the repo and the docs. This is just an initial pass to swap the terminology out, it'll likely be helpful to reorganize the docs a bit as well.

### CODE REVIEW

File: docs/source/core/extracting.rst [hunk 1]
```diff
@@ -270,13 +270,13 @@ Examples:
...
 
-For more information about the ``replace_unicode_quotes`` brick, you can check the `source code here <https://github.com/Unstructured-IO/unstructured/blob/a583d47b841bdd426b9058b7c34f6aa3ed8de152/unstructured/cleaners/core.py>`__.
+For more information about the ``replace_unicode_quotes`` function, you can check the `source code here <https://github.com/Unstructured-IO/unstructured/blob/a583d47b841bdd426b9058b7c34f6aa3ed8de152/unstructured/cleaners/core.py>`__.
 
 
 ``translate_text``
 ------------------
 
-The ``translate_text`` cleaning bricks translates text between languages. ``translate_text``
+The ``translate_text`` cleaning functions translates text between languages. ``translate_text``
```
Comment (qued) [inline docs/source/core/extracting.rst:279]:
```suggestion
The ``translate_text`` cleaning function translates text between languages. ``translate_text``
```

File: docs/source/index.rst [hunk 2]
```diff
@@ -14,8 +14,9 @@ Library Documentation
 :doc:`api`
   Access all the power of ``unstructured`` through the ``unstructured-api`` or learn to host it locally.
 
-:doc:`bricks`
-  Learn more about partitioning, cleaning, and staging bricks, including advanced usage patterns.
+:doc:`core`
+  Learn more about the core partitoning, chunking, cleaning, and staging functionality within the
```
Comment (qued) [inline docs/source/index.rst:18]:
```suggestion
  Learn more about the core partitioning, chunking, cleaning, and staging functionality within the
```

**Review by qued — APPROVED** [review]
Some typos, but overall LGTM!

File: README.md [hunk 3]
```diff
@@ -180,19 +180,25 @@ This README overviews how to install, use and develop the library. For more comp
...
+The easiest way to parse a document in unstructured is to use the `partition` function. If you use `partition` function, `unstructured` will detect the file type and route it to the appropriate file-specific partitioning function. If you are using the `partition` function, you may need to install additional parameters via `pip install unstructured[local-inference]`. Ensure you first install `libmagic` using the instructions outlined [here](https://unstructured-io.github.io/unstructured/installing.html#filetype-detection) `partition` will always apply the default arguments. If you need advanced features, use a document-specific partioning function.
```
[the remark this answers is not in the document]
Comment (ryannikolaidis) [reply README.md:201]: additional very tiny nit on top of alan's suggestion
```suggestion
The easiest way to parse a document in unstructured is to use the `partition` function. If you use the `partition` function, `unstructured` will detect the file type and route it to the appropriate file-specific partitioning function. If you are using the `partition` function, you may need to install additional parameters via `pip install unstructured[local-inference]`. Ensure you first install `libmagic` using the instructions outlined [here](https://unstructured-io.github.io/unstructured/installing.html#filetype-detection) `partition` will always apply the default arguments. If you need advanced features, use a document-specific partioning function.
```

File: examples/arxiv-topic-modelling/topic_model.ipynb [hunk 4]
```diff
@@ -192,9 +192,9 @@
     "stop_words = set(stopwords.words(\"english\"))\n",
     "\n",
     "\n",
-    "# Function to Apply Whatever Cleaning Brick Functionality to Each Narrative Text Element\n",
-    "def custom_clean_brick(narrative_text: str) -> str:\n",
-    "    \"\"\"Apply Mix of Unstructured Cleaning Bricks With Some Custom Functionality to Pre-Process Narrative Text\n",
+    "# Function to Apply Whatever Cleaning Function Functionality to Each Narrative Text Element\n",
```
Comment (ryannikolaidis) [inline examples/arxiv-topic-modelling/topic_model.ipynb:195]: Cleaning Function Functionality? 
```suggestion
    "# Function to Apply Whatever Cleaning Functionality to Each Narrative Text Element\n",
```
or 
```suggestion
    "# Function to Apply Cleaning Functions to Each Narrative Text Element\n",
```
?

**Pull request merged**